### ISSUE TITLE
Werte in kurzen Graphbalken Top 10 

### ISSUE THREAD

**Issue by Zeppelin-PV** [solectrus/solectrus#1822]
hallo Georg
in den Top 10 Listen sowohl aufsteigend als auch absteigend werden bei kurzen Graphbalken keine Werte angezeigt.
Ist es möglich die Werte rechts des Balkens anzuzeigen ?
Die Skalierung der Balken müsste ggf. geändert werden, so dass bei allen immer rechts davon Platz ist.
... einfach so ne Idee
LG Ernst

**Comment by ledermann** [comment]
Ja, korrekt, das kann verbessert werden. Bei kleinen Balken Zahl rechts daneben, bei großen Werten Zahl im Balken (wie jetzt). Mache ich bei Gelegenheit.

**Comment by ledermann** [comment]
Die Werte in der Top-10 werden jetzt je nach Größe des Balkens entweder innerhalb oder außerhalb angezeigt. 
Die Änderung wird im nächsten regulären Update enthalten sein.

**Comment by ledermann** [comment]
Die soeben veröffentlichte Version v0.10.0 enthält die Änderung.
